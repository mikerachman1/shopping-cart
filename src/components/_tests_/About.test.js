import React from "react";
import { render } from "@testing-library/react";

import About from "../About";

describe('About component', () => {
  it('renders the page correctly', () => {
    const { container } = render(<About />);
    expect(container).toMatchSnapshot();
  });
});