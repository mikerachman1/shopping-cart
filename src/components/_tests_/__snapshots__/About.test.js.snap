// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders the page correctly 1`] = `
<div>
  <div
    class="about"
  >
    <div
      class="about-info"
    >
      <h1>
        About Us
      </h1>
      <p>
        Shipper-Shopper is an award winning small company that has been delivering top notch products for over 30 years. Order your stuff from us today and we promise you a wonderful experience. So get shopping!!
      </p>
      <img
        alt="warehouse"
        src="https://imageio.forbes.com/specials-images/imageserve/5f91b847fe7c717493ca4a3b/Warehouse/960x0.jpg?format=jpg&width=960"
      />
    </div>
  </div>
</div>
`;
